internal/pkg/application/functions/registry.go

```diff
@@ -46,7 +46,7 @@ func NewRegistry(ctx context.Context, input io.Reader, storage database.Storage)
 		tokens := strings.Split(line, ";")
 		tokenCount := len(tokens)
 
-		if tokenCount >= 4 {
+		if tokenCount >= 5 {
```

Comment:
I would prefer if the onupdate column was made optional (handle tokenCount >=4 here as before), so that we don't have to add ;false to all our functions. Add a check on line 55 instead to only dereference the fifth element if there are enough tokens.